fossilize_external_replayer_linux.hpp

```diff
@@ -49,10 +49,59 @@
 #include "platform/gcc_clang_spinlock.hpp"
 #endif
 
+#ifdef __linux__
+
 #ifndef SCHED_BATCH
 #define SCHED_BATCH 3
 #endif
 
+/**
+ * Define the syscall interface in Linux because it is missing from glibc
+ * https://www.kernel.org/doc/html/latest/block/ioprio.html
+ */
+
+#ifndef IOPRIO_CLASS_SHIFT
+#define IOPRIO_CLASS_SHIFT (13)
+#endif
+
+#ifndef IOPRIO_PRIO_MASK
```

Comment:
Is this define necessary?

Reply:
Probably not... It is more or less copy and paste from the kernel and/or gamemoded. But Fossilize only uses a subset of the functions provided by those defines. I'll clean that up.